api/automoderation.go

```diff
@@ -0,0 +1,51 @@
+package api
+
+import (
+	"github.com/diamondburned/arikawa/v3/discord"
+	"github.com/diamondburned/arikawa/v3/utils/httputil"
+)
+
+func (c *Client) ListAutoModerationRules(guildID discord.GuildID) ([]discord.AutoModerationRule, error) {
```

Comment:
Could you put copy-pasted Discord API documentation comments here for convenience?

Reply:
Done in [35c61e8](https://github.com/diamondburned/arikawa/pull/444/commits/35c61e83a30d37a5d42e43af925be5537893ff04)